src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+const personStatus = (sex) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return partner;
+};
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    <p className={`${person.age ? `Person__age` : ''}`}>
```

Comment:
You have empty p tag, try to avoid this
![image](https://github.com/mate-academy/react_person/assets/54936044/1e3cd4bd-798f-4ea4-9167-2382fe95bc1c)